glossary.yml

```
@@ -4093,7 +4097,13 @@
     term: ውሸት
     def: >
       የ ሎጂካዊ ([ቦሊያን]ሁኔታ ከ [እውነተኛ](#boolean) ውስጥ ጥቅም ላይ ውሏልየ [ሁለትዮሽ](#true) ሁኔታን ለመወከል አመክንዮ እና መርሃግብር አንድ ነገር ::
-
+      
+   sw:
+    term: "uongo"
+    def: >
+      Mantiki ya [Boolean](#boolean) iliyo katika hali kinyume cha [kweli](#true). 
+      Inatumika katika mantiki na upangaji kuwakilisha hali [binary](#binary) ya kitu.
```

This key was off by one space.

```suggestion
  sw:
    term: "uongo"
    def: >
      Mantiki ya [Boolean](#boolean) iliyo katika hali kinyume cha [kweli](#true). 
      Inatumika katika mantiki na upangaji kuwakilisha hali [binary](#binary) ya kitu.
```